subprojects/docs/src/samples/userguide/tasks/incrementalTask/incrementalTaskNoChange.out

```diff
@@ -0,0 +1,4 @@
+> Task :incrementalReverse UP-TO-DATE
```

Comment:
@pledbrook I added some output for the up-to-date sample. WDYT?

Reply:
Looks good, but the preceding text should also be updated as it mentions the lack of output (which is no longer lacking).